src/odb/test/cpp/TestCallBacks.cpp

```diff
@@ -58,11 +58,13 @@ BOOST_AUTO_TEST_CASE(test_inst_and_iterm)
   BOOST_TEST(cb->events[1] == "PostMove inst i1");
   cb->clearEvents();
   i1->setOrigin(100, 100);
-  BOOST_TEST(cb->events.size() == 0);
-  i1->findITerm("a")->connect(n1);
   BOOST_TEST(cb->events.size() == 2);
-  BOOST_TEST(cb->events[0] == "PreConnect iterm to net n1");
-  BOOST_TEST(cb->events[1] == "PostConnect iterm to net n1");
+  BOOST_TEST(cb->events[0] == "PreMove inst i1");
+  BOOST_TEST(cb->events[1] == "PostMove inst i1");
+  i1->findITerm("a")->connect(n1);
```

Comment:
Add a `cb->clearEvents();` before connect to separate events between setOrigin and connect.